test/blockchain.js

```diff
@@ -130,7 +124,6 @@ describe('Verify block of precommits', function () {
             'propose_hash': '1783d20a053b5c45b40e76358a51a7fce90eea391a409decfb9f9cbbb5a4875a',
             'block_hash': '1783d20a053b5c45b40e76358a51a7fce90eea391a409decfb9f9cbbb5a4875a'
           },
-          'network_id': 0,
```

Comment:
Off-topic: why are the object fields quoted here?

Reply:
good catch, thanks